README.md

```diff
@@ -1,172 +1,31 @@
 # Keboola as Code
 
-- Keboola as Code is **pull/push client** using [Keboola Storage API](https://developers.keboola.com/integrate/storage/api/).
-- It syncs all [component configurations](https://help.keboola.com/components/) from a [KBC](https://www.keboola.com/) project to a directory, and vice versa.
-- The directory can be versioned in [Git](https://git-scm.com/) or a similar tool.
+A monorepo written in Go, represents [Keboola Connection](https://www.keboola.com/product/overview) project as definition files.
 
-## Installation
+## Overview
 
-This console tool is distributed in the following ways.
+### CLI
 
-### MacOS
+- Bidirectional synchronization of a local directory and [Keboola Connection](https://www.keboola.com/product/overview) project:
+  - Push / pull operations.
+  - Calls [Keboola Storage API](https://developers.keboola.com/integrate/storage/api/).
+  - Local directory can be easly versioned by [Git](https://git-scm.com/) or a similar tool.
+- Configurations are represented as JSON files.
+- Transformations are represented as native files, for example SQL, Python, etc.
+- Read more in the [Documentation](https://developers.keboola.com/cli/).
+- CLI entrypoint: [cmd/kbc/main.go](https://github.com/keboola/keboola-as-code/blob/main/cmd/kbc/main.go)
 
-#### [Homebrew](https://brew.sh/)
+### Date Templates
```

Comment:
Data 🙂 (nebo spíš něco jako Project Templates?)

Reply:
tak to vola produkt :man_shrugging: , ale tiez mi to pride mimo

Takto vyzera stara verzia:
![image](https://user-images.githubusercontent.com/19371734/152151415-a3dd797e-9291-4084-9ba0-4a8b756cdd77.png)


Dam iba `Templates` :+1:

Reply:
Aha jo. 😅